src/cs/Ssh/Services/SshService.cs

```diff
@@ -101,7 +101,7 @@ protected async Task SendMessageAsync(SshMessage message, CancellationToken canc
 	/// an implementation of this method may add any event-handlers to the
```

Comment:
Please update the xml doc comments on the changed APIs.

Reply:
I updated comments in the event-args class.

The change in this file is reverted now though. The new revision is simpler overall and more consistent with how other response messages are handled.